ffcx/codegeneration/numba/file_template.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2025 Chris Richardson and Paul T. Kühner
+#
+# This file is part of FFCx.(https://www.fenicsproject.org)
+#
+# SPDX-License-Identifier:    LGPL-3.0-or-later
+"""Template for file output."""
+
+declaration_pre = """
+"""
+
+declaration_post = ""
+
+implementation_pre = """
+# This code conforms with the UFC specification version {ufcx_version}
+# and was automatically generated by FFCx version {ffcx_version}.
+#
+# This code was generated with the following options:
+#
+{options}
+
+import numba
+import numpy as np
+import math
+
+# ufcx enums
+interval = 10
```

Comment:
We should find a way to ensure that these are consistent cross implementation backend